superset-frontend/src/views/CRUD/data/dataset/AddDataset/Footer/index.tsx

```diff
@@ -17,7 +17,65 @@
  * under the License.
  */
 import React from 'react';
+import Button from 'src/components/Button';
+import { t } from '@superset-ui/core';
+import { useSingleViewResource } from 'src/views/CRUD/hooks';
+import withToasts from 'src/components/MessageToasts/withToasts';
+import { DatasetObject } from '../types';
 
-export default function Footer() {
-  return <div>Footer</div>;
+interface FooterObject {
+  url: string;
+  addDangerToast: () => void;
+  datasetObject?: Partial<DatasetObject> | null;
+  onDatasetAdd?: (dataset: DatasetObject) => void;
 }
+
+function Footer({ url, datasetObject, addDangerToast }: FooterObject) {
```

Comment:
I just realize we have two `Footer` components:
`superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/Footer/Footer.tsx`
`superset-frontend/src/views/CRUD/data/dataset/AddDataset/Footer/index.tsx`

Isn't weird to have two components called`<Footer>`  with different `FooterProps` in the same codebase? Should we consider a rename for one of them?

Reply:
I don't think it is that weird. One of these is the footer for the native Filter the other is for AddDataset. I think that the folder structure helps differentiate them. How would you rename them?

Reply:
Well, it depends, if we are keeping them separately like two individual `Footer`s, I'd probably Call them `FilterFooter` and `DatasetFooter` or similar, so I know which `Footer` I'm working with. But yeah, the path is descriptive as you mention, not a blocker, more likely just personal taste, so nvm.